applications/matlab-batch-express/tapisjob_app.sh

```diff
@@ -0,0 +1,51 @@
+# Matlab batch wrapper version R2022a for DesignSafe on VM ds-exec-01 - updated Jul 2022
+handle_error() {
+  local EXITCODE=$1
+  echo "Matlab job exited with an error status. $EXITCODE" >&2
+  echo "$EXITCODE" > "${_tapisExecSystemOutputDir}/tapisjob.exitcode"
+  exit $EXITCODE
+}
+
+set -x
+echo "TACC: job $_tapisJobUUID execution at: `date`"
+
+# set up license file
+cat << EOT >> .matlab_license
+${_license}
+USE_SERVER
+EOT
+
+cat .matlab_license
+export LM_LICENSE_FILE=`pwd`/.matlab_license
+echo "LM_LICENSE_FILE is : $LM_LICENSE_FILE"
+
+cd ${workingDirectory}
+
+echo "Directory is `pwd`"
+echo "File is ${matlabScriptName}"
+formattedInputFile="$( basename ${inputfile} .m )"
```

Comment:
What is this for out of curiosity?

Reply:
The mathworks matlab image directs usage of matlab scripts without the `.m` extension - tried keeping the extension there and it breaks.